### PR TITLE
fix ngui readme

### PR DESCRIPTION
**Pull request by gaarf**
- update readme with `gulp distribute` instructions
- bump version dependencies
- disable some annoyingly failure-prone e2e tests
- clean up `server.js`
- other small things

### CODE REVIEW

File: coopr-ngui/package.json [hunk 1]
```diff
@@ -8,7 +8,9 @@
   },
   "scripts": {
     "start": "node ./server.js",
-    "build": "npm install && ./node_modules/bower/bin/bower update && node ./node_modules/gulp/bin/gulp.js build",
+    "backend": "COOPR_USE_DUMMY_PROVISIONER=true COOPR_DISABLE_UI=true ../coopr-standalone/target/coopr-0.9.9-SNAPSHOT-standalone/bin/coopr.sh",
```
Comment (wolf31o2) [inline coopr-ngui/package.json:11]: What's this one, for? It has a hard-coded PATH to the standalone `coopr.sh` script, which **will** move between releases. That cool?

Comment (gaarf) [inline coopr-ngui/package.json:11]: yeah it's just for convenience.

**Comment by gaarf** [comment]
Looks like Travis is experiencing bad weather again :-I

**Comment by gaarf** [comment]
:8ball: passed, ran for 15 min 55 sec

**Comment by ajainarayanan** [comment]
:+1: LGTM.

**Pull request merged**